### DISCUSSION TITLE
Multiple projects on same machine

### DISCUSSION THREAD

**Discussion by gponty** [dunglas/symfony-docker#529] [Q&A]
Hi,

Is it possible to have several projects on the same machine without having to stop docker on one to restart it on another? If I currently do this, it obviously puts me in "port already busy" mode.

thank you

**Comment by maxhelias** [comment]
Hello,

You have different way to do that : 
- Use different mapping ports for each of your projects.
- Add a new configuration on caddyfile like this one : https://github.com/dunglas/symfony-docker/blob/main/frankenphp/Caddyfile#L16-L55
